### ISSUE TITLE
Fix 3 phase in the electrical room

### ISSUE THREAD

**Issue by JasperWallace** [londonhackspace/hackneyroad#5]
The 3 phase cables into the dist board out side the electrical room are a mess, also the emontx needs connecting up to them properly.

**Comment by russss** [comment]
Yes, this needs taking up to regs. I don't believe the cable currently used to the distribution board meets regulations, either. It at least needs a bit more galvanised trunking.

**Comment by deanforbes** [comment]
I will have a look at this - I expect it to be self explanatory but could some one show me what concerns them - I am aware that there is an inapproriate breaker in the main board 25am D curve I believe supplying the sub board

**Comment by JasperWallace** [comment]
For the power monitoring stuff we could also do with a map of which phases are used on which circuits. (Would also help clarify the mix of old and new wiring).,

**Comment by russss** [comment]
Tgreer and Paddy are aiming to do this on December 14th, I'll check the plans are good before then.

**Comment by padski** [comment]
The required materials are ordered, and the work is scheduled for Monday 14th Dec

**Comment by padski** [comment]
Major part done. Need to switch phases for polisher + possibly dirty workshop supplies
